package.json

```diff
@@ -2,6 +2,9 @@
   "name": "async",
   "description": "Higher-order functions and common patterns for asynchronous code",
   "main": "lib/async.js",
+  "files": [
+    "lib"
+  ],
```

Comment:
Agreed overall, but does npm support folders in `files` or do you specify exact path>?